### DISCUSSION TITLE
App e servizi

### DISCUSSION THREAD

**Discussion by Forrest-It** [iisgiua/giuaschool#383] [Richieste di aiuto]
Vorrei abilitare, dalla pagina "App e Servizi" , la possibilità di scaricare l'App Giua App : è possibile farlo ? 

Grazie

Tommaso

**Comment by trinko** [comment]
La cosa più semplice è aggiungere direttamente un link nel template della pagina.
Lo trovi in **templates/app/info.html.twig**

Dopo le modifiche, svuota la cache, come amministratore, da **SISTEMA > MANUTENZIONE > SVUOTA CACHE**

**Comment by Lucaccc** [comment]
> Vorrei abilitare, dalla pagina "App e Servizi" , la possibilità di scaricare l'App Giua App : è possibile farlo ?
> 
> Grazie
> 
> Tommaso

Salve, le faccio io una domanda ora: che hosting sta usando per il registro?
Vorrebbe darmi una mano nella reinstallazione del programma, perchè ho avuto un po di problemi dopo l'aggiornamento.
Mi faccia sapere così le lascio la mail o un contatto.

**Comment by Forrest-It** [comment]
Salve,

ho un hosting su Aruba Business : no problem se posso aiutarla in qualche
modo, mi fa piacere.

Che tipo di problema ha ?

Mi faccia sapere

Tommaso

Il giorno mer 5 ott 2022 alle ore 15:51 Lucaccc ***@***.***>
ha scritto:

**Reply by Lucaccc** [reply]
> Salve, ho un hosting su Aruba Business : no problem se posso aiutarla in qualche modo, mi fa piacere. Che tipo di problema ha ? Mi faccia sapere Tommaso Il giorno mer 5 ott 2022 alle ore 15:51 Lucaccc ***@***.***> ha scritto:
> […](#)
> Vorrei abilitare, dalla pagina "App e Servizi" , la possibilità di scaricare l'App Giua App : è possibile farlo ? Grazie Tommaso Salve, le faccio io una domanda ora: che hosting sta usando per il registro? Vorrebbe darmi una mano nella reinstallazione del programma, perchè ho avuto un po di problemi dopo l'aggiornamento. Mi faccia sapere così le lascio la mail o un contatto. — Reply to this email directly, view it on GitHub <[#383 (comment)](https://github.com/iisgiua/giuaschool/discussions/383#discussioncomment-3805450)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ARSISJB76YWMVG5J3IC23ILWBWBX5ANCNFSM6AAAAAAQ3PI2B4> . You are receiving this because you authored the thread.Message ID: ***@***.***>

Ha dimistichezza con altervista? Io ho utilizzato quello fino allo scorso anno scolastico, ma ora mi sta dando problemi.
Carico tutti i file, anche cambiando le info sul file .env (spero che lo faccio nel modo corretto), ma poi non riesco a raggiungere la pagina /install/index.php

**Reply by Forrest-It** [reply]
Salve,

per capire cosa succede avrei bisogno dei dati di accesso:  diversamente le posso solo suggerire qualche controllo da fare.
Magari può iniziare ad inviarmi il file .env

**Reply by Lucaccc** [reply]
> Salve,
> 
> per capire cosa succede avrei bisogno dei dati di accesso: diversamente le posso solo suggerire qualche controllo da fare. Magari può iniziare ad inviarmi il file .env

Se mi lascia una mail le mando tutti i dettagli; altrimenti posso lasciargliela io. Mi faccia sapere.

**Reply by Forrest-It** [reply]
Mi può inviare  i dettagli a tommaso.feola@ivylogic.it

Saluti

**Reply by Lucaccc** [reply]
Le ho inviato la mail.
